src/elements/content-preview/ContentPreview.js

```diff
@@ -686,6 +690,26 @@ class ContentPreview extends React.PureComponent<Props, State> {
         return !!showAnnotations && (this.canAnnotate() || hasViewAllPermissions || hasViewSelfPermissions);
     }
 
+    /**
+     * Returns whether a preview viewer supports printing
+     *
+     * @return {boolean}
```

Comment:
Accurate? We generally leave JSDocs off of new code due to their tendency to become outdated.